client/js/bundles/core/publicParagraphDocument.js

```diff
@@ -16,7 +16,7 @@ import DpPublicDetailNoMap from '@DpJs/components/statement/DpPublicDetailNoMap'
 import { DpUploadFiles } from '@demos-europe/demosplan-ui'
 import { initialize } from '@DpJs/InitVue'
 import publicStatement from '@DpJs/store/statement/PublicStatement'
-import TocStateMemorizer from '@DemosPlanCoreBundle/lib/TocStateMemorizer'
+import TocStateMemorizer from '@DpJs/lib/core/TocStateMemorizer'
```

Comment:
now, as I see this. We should move it to `@DpJs/lib/statement/TocStateMemorizer` not to core

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/614/commits/61b39ed68b497a3abd3098f16dc0a0f74ac69159